### DISCUSSION TITLE
ROCCAT "Burst Pro Air" support

### DISCUSSION THREAD

**Discussion by vladosix** [X3n0m0rph59/eruption#189] [Hardware Support]
Hi, 

Wondering if there could be a support for Burst Pro Air wireless mouse if not already implemented.

Best wishes,

Vlad

**Comment by X3n0m0rph59** [comment]
Hi @vladosix, we currently do not have support for the `ROCCAT Burst Pro Air` (wireless mouse).

If you own such a device and are willing to help, please find instructions here: [Eruption Wiki](https://github.com/X3n0m0rph59/eruption/wiki)
